### DISCUSSION TITLE
Clickhouse sharding support in community version

### DISCUSSION THREAD

**Discussion by slava-val-al** [uptrace/uptrace#325] [Q&A] [answered]
Hello, guys!

Currently, I'm working on introducing uptrace into us production environment and I'm worrying about scaling capabilities. As it wrote in [documentation](https://uptrace.dev/get/config.html#scaling) uptrace doesn't support horizontal scaling in community edition(only in EE one). Are there any plans in roadmap to implement clickhouse sharding support in community edition?

**Comment by vmihailenco** [comment, marked as answer]
There are no such plans in the roadmap. The [documentation](https://uptrace.dev/get/config.html#scaling) explains why vertical scaling is the way to go with the Community edition.
